cli/Fixtures/ios_app_with_framework_headers_in_buildable_folders/Project.swift

```diff
@@ -0,0 +1,49 @@
+import ProjectDescription
+
+let project = Project(
```

Comment:
I'll update this to something more meaningful that uses the headers at compile-time.